### ISSUE TITLE
[BUG] loadConfig never uses cache

### ISSUE THREAD

**Issue by rpereira-anchor** [francoismassart/eslint-plugin-tailwindcss#343]
**Describe the bug**
I have an issue where a change to the way I set my `tailwind.config` file started making timeouts when linting. Loading the actual application is fine.
The change was to determine which local packages my app depends on in order to add it to the content that tailwind has to look into.

When debugging I came to a finding that the tailwind.config file was being loaded more times than should be necessary. One time per file it seems.

It seems to me that, when loading tailwind config, the cache is never used. In [this](https://github.com/francoismassart/eslint-plugin-tailwindcss/blob/master/lib/util/customConfig.js#L42) line, 2 different instances of dates are compared. These are always different.

**To Reproduce**
Place a `console.log` inside each if statement
```js
      if (stats === null) {
        loadedConfig = {};
         console.log("1")
      } else if (lastModifiedDate !== stats.mtime.getTime()) {
        lastModifiedDate = stats.mtime.getTime();
        loadedConfig = requireUncached(resolvedPath);
         console.log("2")
      } else {
        loadedConfig = null;
         console.log("3")
      }
```
only `2` will appear

**Expected behavior**

I expect  `2` to appear once followed by many instances of `3`. This means that the last modified date has not changed.

**Screenshots**
N/A

**Environment (please complete the following information):**
 - OS: Debian 11
 - Softwares + version used:
     -  yarn 4
     -  node 18
     - eslint-plugin-tailwindcss 3.17.0

**Additional context**
N/A

**eslint config file or live demo**
https://github.com/francoismassart/eslint-plugin-tailwindcss/pull/344

**Comment by rpereira-anchor** [comment]
@francoismassart this should be a trivial fix. The PR now compares unix times. I'll write tests if necessary.

**Comment by francoismassart** [comment]
Hi @rpereira-anchor,
Thank you for your contribution, I'm working on a new beta version which fixes this issue:
https://github.com/francoismassart/eslint-plugin-tailwindcss/blob/fix/config-and-css-expiration/lib/util/customConfig.js#L47

And it also avoid re-parsing unchanged CSS files...

You can already test it out via

`npm i eslint-plugin-tailwindcss@3.17.3-beta.2`

**Comment by francoismassart** [comment]
Should be fixed in latest release (3.17.4)

**Comment by rpereira-anchor** [comment]
@francoismassart I have a deadline soon, I'll report back on the fix. Thank you very much!

**Comment by rpereira-anchor** [comment]
@francoismassart it works!
